vkrunner/vr-box.c

```diff
@@ -25,11 +25,11 @@
 
 #include "config.h"
 
-#include "vr-box.h"
-
 #include <assert.h>
 #include <math.h>
 
+#include "vr-box.h"
+#include "vr-test.h"
```

Comment:
It seems a bit messy that there is now a circular dependency between vr_box, vr_test and vr_script. Perhaps it would be cleaner to put the tolerance struct in its own header in the vr_tolerance namespace.